### PR TITLE
fix angles on Bu for use by CICE

### LINKED ISSUE #921: rotation angles on corner grid points are wrong in cpld_gridgen
Since MOM6 creates the model grid at runtime (including adjusting the land mask, if required), files required or used for the global applications for CICE and ATM must be created in a pre-processing step using only the MOM6 supergrid, topography and land mask files as input. This allows the mapped ocean mask (used for creating of the ATM ICs) and the CICE6 grid and mask files to be consistent with the run-time configuration of MOM6. A version of the cpld_gridgen utility has used to generate the required files even prior to being incorporated into ufs_utils, dating back at least 4 years when the authoritative repo was ufs-s2s. 

For CICE, one of the required fields is the rotation angle for vector fields oriented I-J to be mapped to E-W and back. For CICE, this rotation angle is defined on the corner grid points, referred to as the ``Bu`` grid by MOM6. MOM6 itself calculates the rotation angle only on the center (``Ct``) grid points. In cpld_gridgen, in order to calculate the ``Bu`` angle, the ``Ct`` angle procedure from MOM6 was modified. This modification has been used since the earliest versions of cpld_gridgen. 

This does not produce the correct rotation angles for CICE. This figure shows the Ct and (-1)*Bu angle in the current code

baseline

A slice through the middle of the Arctic shows the issue:
Screenshot 2024-03-15 at 1 58 43 PM

The ``Bu`` angle (black) and ``Ct`` angles should be of the same magnitude, since they are simply at slightly different geographic locations. But ``Bu`` is too small away from the tripole seam. For the 1/4deg tripole, the maximum angular error is ~20deg.

The fix is to calculate the ``Bu`` angle using an analogous procedure to that used internally by CICE to go from the ``Bu`` to the ``Ct`` angles. This procedure is shown as

      !-----------------------------------------------------------------
      ! Compute ANGLE on T-grid
      !-----------------------------------------------------------------
      if (trim(grid_type) == 'cpom_grid') then
         ANGLET(:,:,:) = ANGLE(:,:,:)
      else if (.not. (l_readCenter)) then
         ANGLET = c0

         !$OMP PARALLEL DO PRIVATE(iblk,i,j,ilo,ihi,jlo,jhi,this_block, &
         !$OMP                     angle_0,angle_w,angle_s,angle_sw)
         do iblk = 1, nblocks
            this_block = get_block(blocks_ice(iblk),iblk)
            ilo = this_block%ilo
            ihi = this_block%ihi
            jlo = this_block%jlo
            jhi = this_block%jhi

            do j = jlo, jhi
            do i = ilo, ihi
               angle_0  = ANGLE(i  ,j  ,iblk) !   w----0
               angle_w  = ANGLE(i-1,j  ,iblk) !   |    |
               angle_s  = ANGLE(i,  j-1,iblk) !   |    |
               angle_sw = ANGLE(i-1,j-1,iblk) !   sw---s
               ANGLET(i,j,iblk) = atan2(p25*(sin(angle_0)+ &
                                             sin(angle_w)+ &
                                             sin(angle_s)+ &
                                             sin(angle_sw)),&
                                        p25*(cos(angle_0)+ &
                                             cos(angle_w)+ &
                                             cos(angle_s)+ &
                                             cos(angle_sw)))
            enddo
            enddo
         enddo

This same procedure can be used to go from ``Ct`` to ``Bu`` with care for the indexing change and with knowing the ``Ct`` angle on the other side of the seam. Implementing this to calculate the ``Bu`` angles from the known MOM6 ``Ct`` angles produces the following

fix

Screenshot 2024-03-15 at 2 09 19 PM

In addition, once the ``Bu`` angles are known, the ``Ct`` angle that will be calculated by CICE (at run time) can again be calculated and that angle compared with the ``Ct`` angles produced by MOM6. This ``angchk`` has been implemented in a feature branch. The feature branch has been tested and shows that

- the ``Ct`` angle produced by cpld_gridgen are O~10-6 or less different than the actual angle values produced by MOM6 at run time and written to the ocean history file. This has always been the case for the ``cpld_gridgen`` utility.
- the ``Ct`` angle calculated using the ``Bu`` angles in ``cpld_gridgen`` (the ``angchk`` array) are O~10-8 different than the actual ``anglet`` values produced by CICE at runtime and written to the ice history file. This shows that the ``Bu`` angles calculated by ``cpld_gridgen`` are now consistent with the ``Ct`` angles from MOM6.

### PR DESCRIPTION
**Pull request by DeniseWorthen**
Remove find_anq routine and instead calculate Bu angles using the Ct angle on the opposite side of the seam.
Add angchk to verify CICE internally calculated Ct angle against both MOM6 and CICE model output
Remove resetting of longitudes to 0:360
Add  tripole:tripole bilinear mapping for creation of CICE IC downscaling and add required Bu<->Ct mapping on tripole grid
Replace existing unit test on ``Bu`` angles by providing 5x5 blocks of input, finding ``anglet``, ``angle`` and ``angchk`` and comparing known (good) output for MOM6 and CICE6. 

## TESTS CONDUCTED: 
If there are changes to the build or source code, the tests below must be conducted. Contact a repository manager if you need assistance.

- [x] Compile branch on all Tier 1 machines using Intel (Orion, Jet, Hera, Hercules and WCOSS2). Done using 9caec86.
- [x] Compile branch on Hera using GNU. Done using 5da3dc6.
- [x] Compile branch in 'Debug' mode on WCOSS2. Done using 5da3dc6.
- [x] Run unit tests locally on any Tier 1 machine. Done on Hercules using 5da3dc6.
- [x] Run relevant consistency tests locally on all Tier 1 machine.

This PR will change baselines for all resolutions.

- The ``tripole`` master file at each resolution will now contain two additional fields, the ``Bu`` angle and the ``angchk`` array. 
- The previous code added an increment of ``360.00`` in longitude to the MOM6 longitudes, which have values from -300 to 60. This was done only for convenience, but it was found to produce a slight anomaly in the right-hand pole longitude value. 
- Additional mapping files are generated in order to address the requirement of downscaling CICE ICs from higher to lower resolutions.

A test baseline was created using a feature branch containing only the additional angle variables and the removal of the 360deg increment. The reg tests were run for the feature/fixangleBu branch against this test baseline. The following files did not compare due to the change in the Bu angle fields.
```
20:Comparing grid_cice_NEMS_mx025.nc........NOT OK
38:Comparing tripole.mx025.nc........NOT OK
58:Comparing grid_cice_NEMS_mx050.nc........NOT OK
74:Comparing tripole.mx050.nc........NOT OK
94:Comparing grid_cice_NEMS_mx100.nc........NOT OK
107:Comparing tripole.mx100.nc........NOT OK
128:Comparing grid_cice_NEMS_mx500.nc........NOT OK
138:Comparing tripole.mx500.nc........NOT OK
```

## DOCUMENTATION:

- Documentation has been updated. The doxygen was generated on Cactus using 5da3dc6. There we no warnings. It was visualized on rzdm and looked good.

## ISSUE: 

- fixes #921

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I have to rebuild my unit test since I'v re-written the routine it tested.

**Comment by GeorgeGayno-NOAA** [comment]
@DeniseWorthen is there anything I can do to help with this PR?

**Comment by DeniseWorthen** [comment]
No, thanks. I'm just trying to get a new unit test running since I've basically replaced the code that the previous unit test was using. I've almost got it working now.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA This PR will require all new baselines for all platforms, but it is ready for review.

**Comment by GeorgeGayno-NOAA** [comment]
Just a minor comment about an unused variable:

```
/work2/noaa/da/ggayno/save/UFS_UTILS.denise/sorc/cpld_gridgen.fd/gen_fixgrid.F90(52): remark #7712: This variable has not been used.   [JJ]
  integer :: ii,jj
----------------^
```

File: tests/cpld_gridgen/ftst_find_angq.F90 [hunk 1]
```diff
@@ -1,108 +1,335 @@
...
+    implicit none
+
+    real(dbl_kind),    intent(in) :: dmax
+    real(dbl_kind),    intent(in) :: tolerance
+    character(len=*),  intent(in) :: msg
+
+    if (dmax .le. tolerance) then
+       print '(a,2f12.8)','SUCCESS! '//trim(msg)//'  ',dmax,tolerance
+    else
+       print '(a,2f12.8)','FAIL! '//trim(msg)//'  ',dmax,tolerance
```
Comment (GeorgeGayno-NOAA) [inline tests/cpld_gridgen/ftst_find_angq.F90:332]: Need to uncomment the 'stop 1' or this test will silently fail.

Comment (DeniseWorthen) [reply tests/cpld_gridgen/ftst_find_angq.F90:332]: Thanks!

**Comment by GeorgeGayno-NOAA** [comment]
@DeniseWorthen do you have a new set of baseline files or should I create them from your branch?

**Comment by DeniseWorthen** [comment]
I need to generate them. I probably have them on hera, but I'd prefer to regenerate.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I've generated the following baselines

WCOSS2: /lfs/h2/emc/stmp/denise.worthen/CPLD_GRIDGEN/BASELINE
HERA: /scratch1/NCEPDEV/stmp4/Denise.Worthen/CPLD_GRIDGEN/BASELINE
HERCULES: /work/noaa/stmp/dworthen/CPLD_GRIDGEN/BASELINE
ORION: /work/noaa/stmp/dworthen/CPLD_GRIDGEN/BASELINE

For Jet, I got a failure the first time which I seemed like a system error? I'm re-running Jet now.

**Comment by DeniseWorthen** [comment]
Baselines on Jet were successful on 2nd try

 /lfs4/HFIP/h-nems/Denise.Worthen/CPLD_GRIDGEN/BASELINE/

**Comment by GeorgeGayno-NOAA** [comment]
> Baselines on Jet were successful on 2nd try
> 
> /lfs4/HFIP/h-nems/Denise.Worthen/CPLD_GRIDGEN/BASELINE/

Great. I will host the files in the official location, then merge.

**Review by GeorgeGayno-NOAA — APPROVED** [review]

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA I've generated the following baselines
> 
> WCOSS2: /lfs/h2/emc/stmp/denise.worthen/CPLD_GRIDGEN/BASELINE HERA: /scratch1/NCEPDEV/stmp4/Denise.Worthen/CPLD_GRIDGEN/BASELINE HERCULES: /work/noaa/stmp/dworthen/CPLD_GRIDGEN/BASELINE ORION: /work/noaa/stmp/dworthen/CPLD_GRIDGEN/BASELINE
> 
> For Jet, I got a failure the first time which I seemed like a system error? I'm re-running Jet now.

@DeniseWorthen - the files on WCOSS2 are not there. Sorry for not grabbing them sooner.

**Comment by DeniseWorthen** [comment]
No problem. I'll recreate the WCOSS2 baselines now.

**Comment by GeorgeGayno-NOAA** [comment]
I tried your branch (at 5da3dc6) and compared against your new baseline files. I get differences on Jet, Orion and Hercules. Hera matched. The differences appear to be insignificant. Example:

```
Variable  Group  Count         Sum      AbsSum          Min         Max       Range        Mean      StdDev
grid_xt   /      55940 6.89025e-12 1.95485e-09 -2.83862e-11 2.84217e-11 5.68079e-11 1.23172e-16 3.37063e-13
grid_yt   /      80487 1.90951e-10 1.36521e-09 -6.53699e-13 3.69482e-13 1.02318e-12 2.37245e-15 2.25855e-14
land_frac /     117499 1.49255e-12 1.58281e-09 -2.19935e-13 3.04312e-13 5.24247e-13 1.27026e-17 1.87205e-14
Variable Group   Count          Sum      AbsSum          Min         Max       Range         Mean      StdDev
yc_b     /      524184 -7.95517e-11  5.9549e-09 -1.97531e-12 3.69482e-13 2.34479e-12 -1.51763e-16 1.65174e-14
xc_b     /      273892 -1.78092e-10 7.60304e-09 -2.83862e-11 2.84217e-11 5.68079e-11 -6.50227e-16 1.55986e-13
area_b   /      731295 -4.94464e-17 1.11258e-12 -1.01464e-16 6.00648e-17 1.61529e-16 -6.76148e-23 2.06626e-18
frac_a   /      697122 -5.54867e-12 7.83354e-09 -3.00016e-12 4.02767e-12 7.02782e-12  -7.9594e-18 2.11015e-14
frac_b   /      516080  1.64512e-12 6.90457e-09 -2.75513e-12 3.67173e-12 6.42686e-12  3.18772e-18 2.34368e-14
S        /     2774260  1.57107e-12 7.13945e-08 -2.93421e-12 3.75988e-12 6.69409e-12  5.66302e-19 4.06384e-14
```

My tests are here:
- Jet /lfs4/HFIP/emcda/George.Gayno/ufs_utils.git/UFS_UTILS.denise/reg_tests/cpld_gridgen
- Orion /work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.denise/reg_tests/cpld_gridgen
- Hercules /work2/noaa/da/ggayno/save/UFS_UTILS.denise/reg_tests/cpld_gridgen

Just want to make sure I am not doing anything wrong. If these differences don't matter, I will use my test files as the official baseline.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA That seems very odd. I think the baseline should match b4b unless there is an uninitialized variable or something similar. I need to make sure there isn't something wrong. I'll take a look at it.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I tried to run the develop branch against the current baselines for cpld_gridgen on Hercules and all failed.

I can't see your baselines because they're in the ``da`` area.

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA That seems very odd. I think the baseline should match b4b unless there is an uninitialized variable or something similar. I need to make sure there isn't something wrong. I'll take a look at it.

Can you compile your branch on Hercules or Orion? I would like to test it.

**Comment by DeniseWorthen** [comment]
This is a current build on hercules using build_all.sh : /work/noaa/nems/dworthen/utils_dw/exec/cpld_gridgen

**Comment by GeorgeGayno-NOAA** [comment]
> This is a current build on hercules using build_all.sh : /work/noaa/nems/dworthen/utils_dw/exec/cpld_gridgen

Did you build with gnu?

**Comment by DeniseWorthen** [comment]
I've been trying to set up a GNU build on hercules since hera is down today. So I probably did overwrite the intel/release build. Sorry. I've rebuilt it now using intel/release.

**Comment by DeniseWorthen** [comment]
I've been able now to test the develop branch against the current baseline on Hercules. I don't know what happened yesterday, but now all files are B4B except for one, for the 1deg case. I can't explain this.

**Comment by GeorgeGayno-NOAA** [comment]
> I've been able now to test the develop branch against the current baseline on Hercules. I don't know what happened yesterday, but now all files are B4B except for one, for the 1deg case. I can't explain this.

Let me fix the links.

**Comment by GeorgeGayno-NOAA** [comment]
> > I've been able now to test the develop branch against the current baseline on Hercules. I don't know what happened yesterday, but now all files are B4B except for one, for the 1deg case. I can't explain this.
> 
> Let me fix the links.

Try it now.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I'm very confused. After you posted to try again after you fixed the links, I re-executed the "link_fixdirs.sh" command and tried to re-run the develop branch. I once again am getting all baselines failing (like yesterday).

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I've pushed a couple of minor fixes from testing w/ GNU on hercules. I've also created a new baseline on hercules at ``/work/noaa/stmp/dworthen/CPLD_GRIDGEN/20240402.BASELINE``. I've run against that baseline twice and was able to reproduce. 

I spot-checked the mx100 and mx050 against the ``ufs_utils.hercules/reg_tests/cpld_gridgen`` files and I do see some differences, but small. Could you see if you have any trouble reproducing the baseline I created?

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA I've pushed a couple of minor fixes from testing w/ GNU on hercules. I've also created a new baseline on hercules at `/work/noaa/stmp/dworthen/CPLD_GRIDGEN/20240402.BASELINE`. I've run against that baseline twice and was able to reproduce.
> 
> I spot-checked the mx100 and mx050 against the `ufs_utils.hercules/reg_tests/cpld_gridgen` files and I do see some differences, but small. Could you see if you have any trouble reproducing the baseline I created?

I am able to reproduce your new baseline! Please recreate your baselines on Orion, Jet, Hera and WCOSS2. Then, I can do a final test., then merge.

**Comment by DeniseWorthen** [comment]
Great. I'm doing one final consistency check and then I'll start the remaining platforms.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I've satisfied myself that the angchk, anglet and angle variables for the new baselines are as expected compared to the current develop branch. So I'll go ahead and start generating on the other platforms.

**Comment by DeniseWorthen** [comment]
Hera baseline : /scratch1/NCEPDEV/stmp4/Denise.Worthen/CPLD_GRIDGEN/BASELINE
Orion baseline : /work/noaa/stmp/dworthen/CPLD_GRIDGEN/BASELINE

**Comment by DeniseWorthen** [comment]
JET: /lfs4/HFIP/h-nems/Denise.Worthen/CPLD_GRIDGEN/BASELINE
WCOSS2: /lfs/h2/emc/stmp/denise.worthen/CPLD_GRIDGEN/BASELINE/

**Comment by GeorgeGayno-NOAA** [comment]
Using 993c670 I am able to reproduce your baselines on Hercules, WCOSS2 and Hera. However, I am still getting those annoying small differences on Jet and Orion. And on Jet, I get occasional seg faults.

Location of your branch:

- Jet: /lfs4/HFIP/emcda/George.Gayno/ufs_utils.git/UFS_UTILS.denise/reg_tests/cpld_gridgen
- Orion: /work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.denise/reg_tests/cpld_gridgen

I moved your baselines here:

- Jet: /lfs4/HFIP/hfv3gfs/emc.nemspara/role.ufsutils/ufs_utils/reg_tests/cpld_gridgen/baseline_data/*.new
- Orion: /work/noaa/nems/role-nems/ufs_utils/reg_tests/cpld_gridgen/baseline_data/*new

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA This is puzzling. Let me do some work on Orion and Jet and see if I can reproduce on those platforms.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA I'm seeing a lot of intermittent issues. For example, on Orion even if the baseline for one resolution passes, the following job, at the next lower resolution, is never submitted. It seems to disappear into the ether. But by launching the rt.sh again, the job succeeds. On Jet, the first time I ran this morning the job hung. 

Right now, I'm testing bumping the memory up on Jet and increasing the wall clock time everywhere. I don't see that either of these should really be necessary, but at least on Jet I've been able to run against the baseline several times now. Is this all down to Rocky8? I'll keep you posted.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA Just to be sure, the mom6 fix files used right now are set in the cpld_gridgen.sh rt.sh script. For example, 
```
export MOM6_FIXDIR=/scratch1/NCEPDEV/global/glopara/fix/mom6/20220805
```

There is a second directory there (``20231219``) but those files should not be any different. I don't think that is causing any issue, but I'm wondering if I should update those to the later directory.

**Comment by DeniseWorthen** [comment]
On Jet, I have been able to reproduce the baseline at least 4 times using

```
diff --git a/reg_tests/cpld_gridgen/rt.sh b/reg_tests/cpld_gridgen/rt.sh
index 70d11c04..63b0fcc9 100755
--- a/reg_tests/cpld_gridgen/rt.sh
+++ b/reg_tests/cpld_gridgen/rt.sh
@@ -108,9 +108,9 @@ TESTS_FILE="$PATHRT/rt.conf"
 export TEST_NAME=

 # for C3072 on hera, use WLCLK=60 and MEM="--exclusive"
-WLCLK_dflt=20
+WLCLK_dflt=35
 export WLCLK=$WLCLK_dflt
-MEM_dflt="--mem=12g"
+MEM_dflt="--mem=16g"
 export MEM=$MEM_dflt

 cd $PATHRT
@@ -170,7 +170,7 @@ elif [[ $target = jet ]]; then
     NCCMP=nccmp
     PARTITION=xjet
     ulimit -s unlimited
-    WLCLK=25
+    WLCLK=35
     SBATCH_COMMAND="./cpld_gridgen.sh"
 fi
```

**Comment by DeniseWorthen** [comment]
I am going to see if I can reproduce everywhere using the bump in memory and time.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA On WCOSS2, I don't think the new baselines haven't been staged correctly. They look to be pointing to the Dec 20 baselines. 

```
lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:34 025 -> 025.8639547
lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:34 050 -> 050.8639547
lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:34 100 -> 100.8639547
lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:35 500 -> 500.8639547
```

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA On WCOSS2, I don't think the new baselines haven't been staged correctly. They look to be pointing to the Dec 20 baselines.
> 
> ```
> lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:34 025 -> 025.8639547
> lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:34 050 -> 050.8639547
> lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:34 100 -> 100.8639547
> lrwxrwxrwx 1 emc.nems nems   11 Apr  5 19:35 500 -> 500.8639547
> ```

Try it now.

**Comment by GeorgeGayno-NOAA** [comment]
I just retested your branch (9caec86) on Jet and Orion. I still get these annoying differences. The location of your checked out branch is:

Jet: /lfs4/HFIP/emcda/George.Gayno/ufs_utils.git/UFS_UTILS.denise/reg_tests/cpld_gridgen
Orion: /work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.denise/reg_tests/cpld_gridgen

I would like to point to your executable from my work space. And I wonder if we are loading different libraries at runtime.

**Comment by DeniseWorthen** [comment]
I've been running a sanity check this morning everywhere using a clean checkout of the branch. I've gotten jet,orion and hercules and hera to pass. I haven't started WCOSS2 yet.

On orion, my latest checkout and test are in:

/work/noaa/marine/dworthen/utils_dw
/work/noaa/stmp/dworthen/CPLD_GRIDGEN/rt_106867

Both Hercules and Orion write tests to the same location. So my latest Hercules test is

/work/noaa/nems/dworthen/utils_dw/
/work/noaa/stmp/dworthen/CPLD_GRIDGEN/rt_1639913

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA So the differences are in the masked ocean mask files. Are we pointing to different orog fix files?

In /work/noaa/stmp/ggayno/CPLD_GRIDGEN/rt_443522/025, grid.nml shows ``fv3dir=/work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.denise/fix/orog'``

which is 
```
ls -l /work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.denise/fix/orog
lrwxrwxrwx 1 ggayno da 43 Feb 15  2023 /work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.denise/fix/orog -> /work/noaa/global/glopara/fix/orog/20220805
```

In my /work/noaa/stmp/dworthen/CPLD_GRIDGEN/rt_106867/025, grid.nml shows
``fv3dir='/work/noaa/marine/dworthen/utils_dw/fix/orog'``

which is
```
ls -l /work/noaa/marine/dworthen/utils_dw/fix/orog
lrwxrwxrwx 1 dworthen marine 43 Apr 10 06:48 /work/noaa/marine/dworthen/utils_dw/fix/orog -> /work/noaa/global/glopara/fix/orog/20231027
```

**Comment by GeorgeGayno-NOAA** [comment]
@DeniseWorthen I was pointing to some wrong 'fix' files. Everything passes on all machines for me. Sorry for the confusion. I will merge.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Able to successfully run the regression tests against the new baselines. Everything looks good. Will merge.

**Comment by DeniseWorthen** [comment]
@GeorgeGayno-NOAA Great. Thanks for working through it with me.

**Pull request merged**